src/tests/ftest/util/telemetry_utils.py

```diff
@@ -72,11 +106,7 @@ class TelemetryUtils():
         "engine_pool_scrubber_csums_total",
         "engine_pool_scrubber_next_csum_scrub",
         "engine_pool_scrubber_next_tree_scrub",
-        "engine_pool_scrubber_prev_duration",
-        "engine_pool_scrubber_prev_duration_max",
-        "engine_pool_scrubber_prev_duration_mean",
-        "engine_pool_scrubber_prev_duration_min",
-        "engine_pool_scrubber_prev_duration_stddev",
+        _gen_stats_metrics("engine_pool_scrubber_prev_duration"),
```

Comment:
If `_flatten` is only used to handle these, you could just use `*`. E.g.
```
>>> my_list = [4,5,6]
>>> print([1,2,3,*my_list,7,8,9])
[1, 2, 3, 4, 5, 6, 7, 8, 9]
```
So to flatten the results from this:
```suggestion
        *_gen_stats_metrics("engine_pool_scrubber_prev_duration"),
```

Reply:
Ah, hmm. Right. I was thinking that there must be some way to do this without a helper, but I've forgotten how to Python. :(